dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -108,6 +115,13 @@
       %button#cancel-button.secondary{style: "float: left"}= t('cancel')
       %button#ok-button{style: "float: right"}= t('okay')
 
+  #unsubmit-dialogcontent{style: "overflow:hidden"}
```

Comment:
while i prefer to leave the display: none CSS inline here, this CSS can be moved into scss :)